src/witan/models/load_data.clj

```diff
@@ -104,7 +109,7 @@
    Output is map with keyword and core.matrix dataset"
   [keyname filepath]
   (->> (apply-rec-coercion {:type keyname} (load-csv filepath))
-       (dataset-after-coercion)
+       dataset-after-coercion
```

Comment:
great thanks, sorry I had forgotten you didn't need a bracket for a fn in a thread, but perhaps you could make it sound like a fn e.g. convert-to-dataset-after-coercion?